src/modules/Verifications/components/Atribute/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Icon from 'src/modules/General/components/Icon';
+import Input from 'src/modules/General/components/Input';
+import SearchDropdown from 'src/modules/General/components/SearchDropdown';
+import variables from 'src/styles/constants/_exports.module.scss';
+
+import css from './index.module.scss';
+import { AttributeProps } from './index.types';
+import { useAttribute } from './useAttribute';
+
+const Attribute: React.FC<AttributeProps> = ({
+  index,
+  options,
+  selectedAttribute,
+  onChangeAttribute,
+  onDeleteAttribute,
+  errors,
+}) => {
+  const {
+    selected,
+    onSelectAttribute,
+    operators,
+    selectedOperator,
+    onSelectOperator,
+    attributeValue,
+    handleChangeValue,
+  } = useAttribute(index, onChangeAttribute, selectedAttribute);
+  return (
+    <div className={css['container']}>
+      <div className={css['container__inputs']}>
+        <SearchDropdown
+          value={selected}
+          options={options}
+          isSearchable
+          onChange={onSelectAttribute}
+          containerClassName="flex-1 w-full"
+          errors={errors && errors['id']?.message ? [errors['id'].message] : undefined}
+        />
+        <SearchDropdown
+          value={selectedOperator}
+          options={operators}
+          isSearchable
+          onChange={onSelectOperator}
+          containerClassName="flex-1 w-full"
+          errors={errors && errors['operator']?.message ? [errors['operator'].message] : undefined}
+        />
+        <Input
+          value={attributeValue}
```

Comment:
why not use `register`?
P.S i want to add for all component controller to be handle by controller and not in uncontrolled way

Reply:
register just setValues for property,
Here I wanna control an attribute values comparing to other attributes of verification, for example, I should check that not a duplicate attribute is entered, so error handling and setting attribute value is done in the parent component,